code/modules/client/verbs/etips.dm

```diff
@@ -0,0 +1,13 @@
+/client/verb/toggle_tips()

```

Comment:
That's just pure html, but what I meant was a dropdown checkbox. Those are the future